modules/adding-to-a-project.adoc

```diff
@@ -15,3 +15,8 @@ navigation menu.
 . At the top of the page, select the name of the project that you want to add to.
 
 . Click on a method for adding to your project, and then follow the workflow.
+
+[NOTE]
+====
+You can also add components to the topology using xref:../../applications/application_life_cycle_management/odc-viewing-application-composition-using-topology-view.adoc#odc-adding-components-to-an-existing-project_viewing-application-composition-using-topology-view[quick search].
```

Comment:
Saying this pains me: Although xrefs in modules work, Vikram is still asking that we not add xrefs to modules. The recommended workaround is to place xrefs in an `.Additional resources` section in the assembly immediately below the include for this module. I'm aware that this workaround decreases usability.